BackgroundTask.py

```diff
@@ -134,20 +133,20 @@ def queryEDSM(self, systems):
                 return names
             except:
                 # ignore. the EDSM call is not required
-                if __debug__: print_exc()
+                print_exc()

```

Comment:
Should also be in debug. But I noticed a rather large stack trace when that check fails.
Maybe we should change it to a single line: "Failed to get information about systems from EDSM``